eng/pipelines/coreclr/templates/exploratory-job.yml

```diff
@@ -0,0 +1,93 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osSubgroup: ''
+  container: ''
+  runtimeVariant: ''
+  testGroup: ''
+  framework: net5.0 # Specify the appropriate framework when running release branches (ie netcoreapp3.0 for release/3.0)
```

Comment:
why is  the framework defaulting to net5.0?

Reply:
let me change to net6.0. Thanks for pointing.